### ISSUE TITLE
Call Keep Not Working

### ISSUE THREAD

**Issue by stephanoparaskeva** [wazo-platform/wazo-react-native-demo#14]
#### It is running well and I can accept calls
<img width="1005" alt="Screenshot 2020-06-24 at 15 56 27" src="https://user-images.githubusercontent.com/40718272/85582104-13cdd580-b635-11ea-9372-b63f7b2ec485.png">

#### Expected 
<img width="532" alt="Screenshot 2020-06-24 at 15 57 14" src="https://user-images.githubusercontent.com/40718272/85582213-26e0a580-b635-11ea-8bbd-116225576a28.png">

#### Actual
<img width="833" alt="Screenshot 2020-06-24 at 16 11 38" src="https://user-images.githubusercontent.com/40718272/85582525-6a3b1400-b635-11ea-9db6-661644d64308.png">

**How does one enable Call Keep to display the expected screen whenever a call occurs?**

**Comment by manuquentin** [comment]
Hi @stephanoparaskeva ,
Callkeep doesn't work on a simulator, it's a restriction from Apple.

**Comment by stephanoparaskeva** [comment]
> Hi @stephanoparaskeva ,
> Callkeep doesn't work on a simulator, it's a restriction from Apple.

Awesome, will callkeep show that Page even when both users are using the app in the foreground?
I want the callkeep `Accept Decline`  Page to always show whether the app is closed and off or whether both users are holding the app and using it?

**Comment by manuquentin** [comment]
Yes this demo shows callkeep when making a call or receiving one.

If the app is closed, you'll have to setup [remote push notification](https://github.com/react-native-webrtc/react-native-callkeep#receiving-a-call-when-the-application-is-not-reachable) as the callkeep doc describres.

**Comment by stephanoparaskeva** [comment]
> If the app is closed, you'll have to setup [remote push notification](https://github.com/react-native-webrtc/react-native-callkeep#receiving-a-call-when-the-application-is-not-reachable) as the callkeep doc describres.

Will this work with notifications in App via AppSync and DynamoDB for Android and IOS?

**Comment by manuquentin** [comment]
I don't know, I'have never used this.

**Comment by stephanoparaskeva** [comment]
Reopening as CallKeep did not work while testing from Simulator to my Physical iPhone
![Uploading IMG-0196.PNG…]()

**Comment by GaganBhat** [comment]
Did you figure out the solution to the issue? @stephanoparaskeva
